.github/actions/deps/action.yaml

```diff
@@ -0,0 +1,35 @@
+---
+name: "Dependencies"
+description: "Install dependencies"
+inputs:
+  toolchain:
+    description: Rust toolchain to use, stable / nightly / beta
+    default: stable
+runs:
+  using: composite
+  steps:
+    - uses: actions-rs/toolchain@v1
```

Comment:
This action is unmaintained, please use `dtolnay/rust-toolchain@master` instead for consistency with our other repos.